recipe/meta.yaml

```
@@ -1,45 +1,58 @@
 {% set name = "sacremoses" %}
-{% set version = "0.0.43" %}
+{% set version = "0.1.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 123c1bf2664351fb05e16f87d3786dbe44a050cfd7b85161c09ad9a63a8e2948
+  url: https://github.com/hplt-project/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 139faad4800a5da04498e4b5c297a5a8e1c958877b59090715d8a3f90e0b9b76
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   number: 0
   entry_points:
     - sacremoses=sacremoses.cli:cli
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
 
 requirements:
   host:
-    - pip
     - python
+    - pip
+    - setuptools
+    - wheel
   run:
-    - click
-    - joblib
     - python
-    - regex
-    - six
-    - tqdm
+    - pip
+    - click ==7.1.2
```

```suggestion
    - click >=7.1.2
```
The code is actually compatible with click 8: https://github.com/hplt-project/sacremoses/commit/01d5c7392b2b0c39f22c6bf12b318e8242cc61da

The constraints from requirements.txt are not applied in their packaging. Instead, this is what is used: https://github.com/hplt-project/sacremoses/blob/master/setup.py#L34

But I am okay to keep the minimum bounds you currently have.

---

Thanks! Yep, since no pinnings in setup.py, thought of at least matching the requirements.txt as baseline.
Adding a comment about it, and removing the exact click pinning.